sse2neon.h

```diff
@@ -2734,6 +2734,23 @@ FORCE_INLINE __m128i _mm_mullo_epi32(__m128i a, __m128i b)
         vmulq_s32(vreinterpretq_s32_m128i(a), vreinterpretq_s32_m128i(b)));
 }
 
+// Multiply the packed unsigned 16-bit integers in a and b, producing
+// intermediate 32-bit integers, and store the high 16 bits of the intermediate
+// integers in dst.
+//
+//   FOR j := 0 to 3
+//      i := j*16
+//      tmp[31:0] := a[i+15:i] * b[i+15:i]
+//      dst[i+15:i] := tmp[31:16]
+//   ENDFOR
+//
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_m_pmulhuw
+FORCE_INLINE __m64 _m_pmulhuw(__m64 a, __m64 b)
```

Comment:
Don't implement duplicated function.
Instead, we can simply use macro:
```c
#define _m_pmulhuw(a, b) _mm_mulhi_pu16(a, b)
```

Reply:
I did not notice that there is another duplicated function.
I think I will rewrite it like the duplicated function `_m_pavgb`
```c
FORCE_INLINE __m64 _m_pavgb(__m64 a, __m64 b)
{
    return _mm_avg_pu8(a, b);
}
```
I am not sure which one is better.
Using macro would be simpler but using function would be more consistent.

Besides, I will change the implementation of `_mm_mulhi_pu16` from
`vmovn_u32 + vshrq_n_u32` to `vshrn_n_u32`.

Reply:
We can implement inline functions starting with `_mm_`, and `_m_pavgb` can be reimplemented as a macro.

Reply:
Got it.
I will do it in the future pull request.